client/src/components/InputField/InputField.js

```diff
@@ -55,7 +77,58 @@ class InputField extends Component {
     }
   }
 
+  handleTipToggle() {
+    this.setState((state) => ({ tipOpen: !state.tipOpen }));
```

Comment:
Right now the tip only goes away when I click on the toggle button. Feels like it should be dismissed when the main field loose focus or when the user hits the ESC key.

Reply:
This was the interaction pattern agreed upon with @clarkepaul, but I could add an additional event if it makes sense (though this would make it tougher to extract into a separate component).

Reply:
Originally I wanted it to open when either the field or button was in focus, naturally that when either was no longer in focus the tooltip would close. After discussing it seemed it was going to complicate things so opted with the simplest form to get it in. If adding the functionality to close the tip when the focus is lost that would be great.

Reply:
I pushed for the limited triggers to make sure the popover behaved appropriately for screenreaders - if it opened as soon as the field had focus, it would trample the announcement for "field gained focus" with the popover's contents. (If we could use passive mode, this wouldn't happen, but that results in the popover's contents being read aloud twice by VoiceOver for some reason).

I'll look into having the popover close when both the field and button have lost focus.

Reply:
I've opted not to pursue the focus-change behaviour, as it would be too time-intensive to find a good pattern for it. We can look into implementing this in the future.